plugins/modules/discovery_intent.py

```diff
@@ -734,8 +749,10 @@ def get_discovery_device_info(self, discovery_id=None, task_id=None):
 
         if not result:
             msg = 'Discovery network device with id {0} has not completed'.format(discovery_id)
+            self.log(msg, "CRITICAL")
             self.module.fail_json(msg=msg)
 
+        self.log('Discovery network device with id {0} has not completed'.format(discovery_id), "INFO")
```

Comment:
if not result, we set the msg with same message (line num: 751) and if the result is true then also we see the same message. Can you please check it?

Reply:
Thanks for pointing it. Resolved